src/modules/aws_ecr@0.0.1/index.ts

```diff
@@ -245,7 +245,7 @@ export const AwsEcrModule: Module = new Module({
     }),
     repositoryPolicy: new Mapper<AwsRepositoryPolicy>({
       entity: AwsRepositoryPolicy,
-      entityId: (e: AwsRepositoryPolicy) => e.repository?.repositoryName + '',
+      entityId: (e: AwsRepositoryPolicy) => e.repository?.repositoryName + '' ?? e.id.toString(),
```

Comment:
that `+ ''` seems like a mistake?

Reply:
Yeah, that'd probably turn into an `'undefined'` string. I can remove it.